packages/frontend/src/build/config/common.ts

```diff
@@ -20,17 +20,13 @@ export const common: Omit<Config, 'backend'> = {
     banner: new Date() <= GITCOIN_BETA_ROUND_END,
     gitcoinOption: false,
     hiring: true,
-    hiringBadge: false,
+    hiringBadge: true,
```

Comment:
Why this change?

Reply:
Oh, my mistake, I thought I discarded it.